base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -320,8 +320,8 @@ impl LMDBDatabase {
     {
         let mut output: TransactionOutputRowData =
             lmdb_get(txn, &self.utxos_db, key).or_not_found("TransactionOutput", "key", key.to_string())?;
-        let result = output.output.clone();
-        output.output = None;
+        let result = output.output.take();
+        // output.output is None
```

Comment:
?

Reply:
Just making it clear :) Maybe too much

Reply:
I like it. Let's use the facilities of the programming language we've chosen.

Reply:
Word! and no clone - I know where the confusion is, commenting out `// output.output = None;` looks very similar to my comment  ;)

Reply:
Yeah I was looking at the rust source, avoiding the clone is the cherry on top 🍒